controllers/component_build_controller_pac.go

```diff
@@ -449,8 +458,11 @@ func (r *ComponentBuildReconciler) ConfigureRepositoryForPaC(ctx context.Context
 	authorName := "redhat-appstudio"
 	authorEmail := "appstudio@redhat.com"
 
+	var pacTargetBranch string
+
 	if component.Spec.Source.GitSource != nil && component.Spec.Source.GitSource.Revision != "" {
 		baseBranch = component.Spec.Source.GitSource.Revision
+		pacTargetBranch = component.Spec.Source.GitSource.Revision
```

Comment:
can we merge this with baseBranch? That would safe us a github/gitlab call.

Reply:
I'm not sure I get your point. Can you be more specific?

Reply:
baseBranch and pacTargetBranch is the same value, when base branch is "" then it's detected, which is exactly what we are doing. So you should be able to use baseBranch and just getDefault branch in this part of code.

Reply:
Done. PTAL.